README.md

```diff
@@ -82,15 +82,15 @@ var format = d3.format(".2s");
 
 ## API Reference
 
-<a name="format" href="#format">#</a> d3.<b>format</b>(<i>specifier</i>)
+<a name="format" href="#format">#</a> d3.<b>format</b>(<i>specifier</i>) [<>](https://github.com/d3/d3-format/blob/master/src/formatDefault.js "Source")
 
 An alias for [*locale*.format](#locale_format) on the [default locale](#formatDefaultLocale).
 
-<a name="formatPrefix" href="#formatPrefix">#</a> d3.<b>formatPrefix</b>(<i>specifier</i>, <i>value</i>)
+<a name="formatPrefix" href="#formatPrefix">#</a> d3.<b>formatPrefix</b>(<i>specifier</i>, <i>value</i>) [<>](https://github.com/d3/d3-format/blob/master/src/formatPrefixAuto.js "Source")
```

Comment:
formatPrefixAuto.js contains the implementation for the `s` format type, as in `d3.format(".3s")`. The correct link for d3.formatPrefix is https://github.com/d3/d3-format/blob/master/src/defaultLocale.js#L5